internal/cmd/config/config.go

```diff
@@ -244,42 +252,47 @@ func DevKeyGeneration() (string, string, string) {
 		panic(err)
 	}
 	if n != numBytes {
-		panic(fmt.Errorf("expected to read 64 bytes, read %d", n))
+		panic(fmt.Errorf("expected to read 32 bytes, read %d", n))
 	}
-	controllerKey := base64.StdEncoding.EncodeToString(randBuf.Bytes()[0:32])
-	workerAuthKey := base64.StdEncoding.EncodeToString(randBuf.Bytes()[32:64])
-	recoveryKey := base64.StdEncoding.EncodeToString(randBuf.Bytes()[64:numBytes])
-
-	return controllerKey, workerAuthKey, recoveryKey
+	devKey := base64.StdEncoding.EncodeToString(randBuf.Bytes())
+	return devKey
 }
 
 // DevController is a Config that is used for dev mode of Boundary
 // controllers
 func DevController() (*Config, error) {
-	controllerKey, workerAuthKey, recoveryKey := DevKeyGeneration()
+	controllerKey := DevKeyGeneration()
```

Comment:
What specifically would I be testing for? That the config created is the same except for the generated keys?

Reply:
You've added an expected outcome which I believe other parts of the implementation rely on.  It might be nice to ensure it's correct.  you might even consider checking that the keys meet some set of constraints, since there are no unit tests for `DevKeyGeneration(...)`.  It's like getting two birds with one stone.

Reply:
Sorry, I should have asked in the first place whether this unit test is sufficiently covered by the `TestDevController` func in `config_test.go`

Reply:
yes it is.